x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -323,6 +370,22 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({ jobId, end,
                         <Settings
                           showLegend
                           legendPosition={Position.Bottom}
+                          onAnnotationClick={(annotations: {
```

Comment:
Are there any privileges checks done here, like how in the Model snapshots tab the action is disabled for users without the correct capabilities?

Reply:
Added in 69bc20da7b898b06e207d35fba64663e7612a829